File: SecondYear/Anantj2499_AnantJaiswal_2226cseml1088_2/Week_1/Data_Types.java
public class Data_Types {
    public static int dataTypes(String type) {
        // Write your code here
        if (type.equalsIgnoreCase("Integer")) {
            return 4;
        }
        if (type.equalsIgnoreCase("Long")) {
            return 8;
        }
        if (type.equalsIgnoreCase("Float")) {
            return 4;
        }
        if (type.equalsIgnoreCase("Double")) {
            return 8;
        }
        if (type.equalsIgnoreCase("Character")) {
            return 1;
        }
        else
            return 0;
    }
}

File: SecondYear/chaudharyaryanpanwar_AryanPanwar_2226cs1053_2/Week_1/dataTypes.cpp
int dataTypes(string type) {
	// Write your code here
    if(type[0] == 'I'){
        return sizeof(int);
    }
    else if(type[0] == 'L'){
        return sizeof(long);
    }
    else if(type[0] == 'F'){
        return sizeof(float);
    }
    else if(type[0] == 'C'){
        return sizeof(char);
    }
    else{
        return sizeof(double);
    }
   return 0;
}
